### PR TITLE
Move PARACHUTE_ACTION enum from ardupilotmega.xml to common.xml (NFC …

### PR DESCRIPTION
**Pull request by amilcarlucas**
…the description was already there)

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1337,7 +1337,7 @@
       </entry>
       <entry value="208" name="MAV_CMD_DO_PARACHUTE">
         <description>Mission command to trigger a parachute</description>
-        <param index="1">action (0=disable, 1=enable, 2=release, for some systems see PARACHUTE_ACTION enum, not in general message set.)</param>
+        <param index="1" label="action" enum="PARACHUTE_ACTION">action</param>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: For consistency capitalise description to "Action".

**Comment by hamishwillee** [comment]
This is great, and I would happily merge this but ...

How do we co-ordinate with downstream ArduPilot? If this were to get pulled in to ardupilot repo before ardupilot.xml got updated then there would be a clash due to the enum being defined twice.

**Comment by amilcarlucas** [comment]
This single commit moves it between the two files. Just merging it downstream will fix it all.

**Comment by hamishwillee** [comment]
Thanks @amilcarlucas - Works for me!

**Pull request merged**